<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AppointmentUi</class>
 <widget class="QWidget" name="AppointmentUi">
  <property name="windowModality">
   <enum>Qt::ApplicationModal</enum>
  </property>
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>254</width>
    <height>378</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout_3">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_2">
       <item>
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Navn</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="appointmentName"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_9">
       <item>
        <widget class="QRadioButton" name="radioAvtale">
         <property name="text">
          <string>Avtale</string>
         </property>
         <property name="checked">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QRadioButton" name="radioFravaer">
         <property name="text">
          <string>Fravær</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <item>
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Start</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeStart"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_6">
       <item>
        <widget class="QLabel" name="label_5">
         <property name="text">
          <string>Slutt</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QDateTimeEdit" name="dateTimeEnd"/>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QFrame" name="frameGjenta">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QCheckBox" name="repeatCheckBox">
            <property name="text">
             <string>Gjenta i (antall uker):</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QSpinBox" name="repeatSpinBox">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame_2">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <property name="margin">
         <number>0</number>
        </property>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QFrame" name="frame">
       <property name="frameShape">
        <enum>QFrame::StyledPanel</enum>
       </property>
       <property name="frameShadow">
        <enum>QFrame::Raised</enum>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <property name="margin">
         <number>0</number>
        </property>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QRadioButton" name="radioUndervisning">
            <property name="text">
             <string>Undervisning</string>
            </property>
            <property name="checked">
             <bool>true</bool>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioLab">
            <property name="text">
             <string>Laboratorium</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QRadioButton" name="radioMote">
            <property name="text">
             <string>Møte</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QRadioButton" name="radioAnnet">
            <property name="text">
             <string>Annet</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="lineEditAnnet">
            <property name="enabled">
             <bool>false</bool>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_4">
       <item>
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Sted</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditLocation"/>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_3">
       <item>
        <widget class="QLabel" name="labelContact">
         <property name="text">
          <string>Kontakter</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QLineEdit" name="lineEditContact">
         <property name="enabled">
          <bool>false</bool>
         </property>
         <property name="readOnly">
          <bool>true</bool>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QToolButton" name="buttonContactsList">
         <property name="text">
          <string>...</string>
         </property>
        </widget>
       </item>
      </layout>
     </item>
     <item>
      <widget class="QTextEdit" name="textEdit"/>
     </item>
     <item>
      <widget class="QDialogButtonBox" name="buttonBox">
       <property name="maximumSize">
        <size>
         <width>16777215</width>
         <height>16777215</height>
        </size>
       </property>
       <property name="standardButtons">
        <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
       </property>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
  <zorder>layoutWidget</zorder>
  <zorder></zorder>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <tabstops>
  <tabstop>appointmentName</tabstop>
  <tabstop>dateTimeStart</tabstop>
  <tabstop>dateTimeEnd</tabstop>
  <tabstop>repeatCheckBox</tabstop>
  <tabstop>repeatSpinBox</tabstop>
  <tabstop>radioAvtale</tabstop>
  <tabstop>radioFravaer</tabstop>
  <tabstop>radioUndervisning</tabstop>
  <tabstop>radioLab</tabstop>
  <tabstop>radioMote</tabstop>
  <tabstop>radioAnnet</tabstop>
  <tabstop>lineEditAnnet</tabstop>
  <tabstop>lineEditLocation</tabstop>
  <tabstop>lineEditContact</tabstop>
  <tabstop>buttonContactsList</tabstop>
  <tabstop>textEdit</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>AppointmentUi</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>136</x>
     <y>395</y>
    </hint>
    <hint type="destinationlabel">
     <x>125</x>
     <y>157</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>repeatCheckBox</sender>
   <signal>toggled(bool)</signal>
   <receiver>repeatSpinBox</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>86</x>
     <y>107</y>
    </hint>
    <hint type="destinationlabel">
     <x>175</x>
     <y>107</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
